crates/sui-config/src/node.rs

```diff
@@ -455,6 +463,17 @@ impl ConsensusConfig {
     pub fn narwhal_config(&self) -> &NarwhalParameters {
         &self.narwhal_config
     }
+
+    pub fn db_retention_epochs(&self) -> u64 {
+        self.db_retention_epochs.unwrap_or(1)
```

Comment:
I don't remember an exact reason, other than being cautious around initial launch. Now we have more experience, it seems the additional history is not too useful for every validator to keep.

Reply:
I cant think of any reasons why we would need to use a past consensus DB, if there is a fork which we need to investigate then it would surface in that epoch. Cant close the epoch if there is any split brain situation.